cmd/bootnode/main.go

```diff
@@ -108,7 +108,7 @@ func main() {
 	logConn := flag.Bool("log_conn", false, "log incoming/outgoing connections")
 	maxConnPerIP := flag.Int("max_conn_per_ip", 10, "max connections number for same ip")
 	forceReachabilityPublic := flag.Bool("force_public", false, "forcing the local node to believe it is reachable externally")
-	noTransportSecurity := flag.Bool("no_transport_security", false, "disable TLS encrypted transport")
+	noTransportSecurity := flag.Bool("no_transport_security", true, "disable TLS encrypted transport")
```

Comment:
I put a comment here to follow up with the rest of 3 cases

Reply:
I would recommend to change this to `false` to follow the current mainnet setting and avoid any potential network disruption